SwiftCompilerSources/Sources/SIL/WitnessTable.swift

```diff
@@ -71,9 +71,8 @@ public struct WitnessTable : CustomStringConvertible, NoReflectionChildren {
                                                      OptionalBridgedFunction(obj: witness?.bridged.obj))
       case .associatedType(let requirement, let witness):
         return BridgedWitnessTableEntry.createAssociatedType(requirement.bridged, witness.bridged)
-      case .associatedConformance(let requirement, let substType, let witness):
```

Comment:
@eeckstein I was lazy but the `substType` tuple element can be removed from this enum case entirely now.

Reply:
Done.